docs/vsphere_ci/build.json

```diff
@@ -1,9 +1,9 @@
 {
   "variables": {
-    "os-iso-path": "[WorkloadDatastore] windows-iso-images/winsrv2004_sep_2020_x64_dvd.iso",
+    "os-iso-path": "[WorkloadDatastore] windows-iso-images/winsrv2022_sep_2021_x64_dvd.iso",
```

Comment:
@saifshaikh48   Does this ISO includes the KB5012637 patch?

Reply:
Then, consider documenting the steps you followed to install the patch. Ideally, it should result in a script executed by Packer during the creation of the VM template.

Reply:
Sure I can document the steps but automating is not trivial. It's not a default-ly accessible patch (there's some name for it, I forget what they called it in sig-windows meeting) so we have to actually download the files from MSFT and then apply it.

Reply:
[Added](https://github.com/saifshaikh48/windows-machine-config-operator/blob/hotfix-golden-image-2022/docs/vsphere-golden-image.md#5-install-os-level-container-networking-patch-kb5012637)

Reply:
Nice! Wondering if a PowerShell script (e.g. install-kb5012637.ps1) under the [scripts directory](https://github.com/saifshaikh48/windows-machine-config-operator/tree/hotfix-golden-image-2022/docs/vsphere_ci/scripts) that downloads the [file](https://catalog.s.download.windowsupdate.com/c/msdownload/update/software/updt/2022/04/windows10.0-kb5012637-x64_6a7459b60e226b0ad0d30b34a4be069bee4d2867.msu) can be included in the [sysprep](https://github.com/saifshaikh48/windows-machine-config-operator/blob/hotfix-golden-image-2022/docs/vsphere_ci/scripts/autounattend.xml#L101) process. Will that capture the automation of the above [steps](https://github.com/saifshaikh48/windows-machine-config-operator/blob/hotfix-golden-image-2022/docs/vsphere-golden-image.md#5-install-os-level-container-networking-patch-kb5012637).